src/platform/shell.go

```diff
@@ -427,19 +427,26 @@ func (env *Shell) HasFiles(pattern string) bool {
 	defer env.Trace(time.Now(), pattern)
 	cwd := env.Pwd()
 	fileSystem := os.DirFS(cwd)
-	matches, err := fs.Glob(fileSystem, pattern)
+	matches, err := fs.ReadDir(fileSystem, ".")
 	if err != nil {
 		env.Error(err)
 		env.Debug("false")
 		return false
 	}
 	for _, match := range matches {
-		file, err := fs.Stat(fileSystem, match)
-		if err != nil || file.IsDir() {
+		if match.IsDir() {
 			continue
 		}
-		env.Debug("true")
-		return true
+		matchFileName, err := filepath.Match(strings.ToLower(pattern), strings.ToLower(match.Name()))
```

Comment:
`strings.ToLower(pattern)` should be done outside of the loop. It can in fact simply be a reassignment of pattern just before the `for` loop.

Reply:
@JanDeDobbeleer It should be resolved in this commit 
https://github.com/JanDeDobbeleer/oh-my-posh/pull/4303/commits/f790e50604e5aa4141336d5ea3a2b2503ca4ca95